src/Enterspeed.Cli/Commands/Deploy/DeployCommands.cs

```diff
@@ -4,15 +4,13 @@ namespace Enterspeed.Cli.Commands.Deploy
 {
     public class DeployCommands
     {
-        public static Command BuildCommands()
+        public static Command[] BuildCommands()
         {
-            var command = new Command("deployment")
+            return new[] { "deployment", "dp" }.Select(commandName => new Command(commandName)
```

Comment:
I don't think this is the right way to add an alias 😄 
Just do `command.AddAlias("dp");`

Reply:
:-o I totally missed it, I looked for it though. Will update